src/core/Akka.Cluster/SBR/SplitBrainResolverProvider.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Akka.Actor;
+using Akka.Coordination;
+
+namespace Akka.Cluster.SBR
+{
+    /// <summary>
+    /// Enabled with configuration:
+    /// {
+    /// akka.cluster.downing-provider-class = "Akka.Cluster.SBR.SplitBrainResolverProvider"
+    /// }
+    /// </summary>
+    public class SplitBrainResolverProvider : IDowningProvider
```

Comment:
I think we ought to make this backwards compatible with what we currently have in our documentation:

```
downing-provider-class = "Akka.Cluster.SplitBrainResolver, Akka.Cluster"
```

Can't really force people to update their configurations during a minor update. We can either resolve this with a branch inside the configuration classes for Akka.Cluster or we can do it via changing this type name. Thoughts?

Reply:
I didn't replace the old resolver, so user can choose, which resolver to use. Because there is one scenario (`keep-referee`) which is not supported in the new resolver.
If you want to completely replace the old provider, we can either change the name & namespace or add some wrapper there or something. And make obsolete `keep-referee` scenario, which is also no longer supported in akka

Reply:
Got it - I'll noodle on that some.